ONSdigital/census-rm-deploy
# Add on_error alerts to pipelines

# Motivation and Context
Add on_error alerts to all jobs in the pipeline so that when a job errors (goes orange), a Slack alert is sent, as it already is with job failures

# What has changed
- Added `on_error` steps to every job that already has an `on_failure` step
- Changed the colours of the `on failure` alerts to #ff0000 (red).  `on_error` alerts will be #f58a3d (orange) - to match the pipeline colours.
- Reformatted the `on_failure` alerts in the pipelines to remove unnecessary whitespace and to keep it tidy.
- Removed unused alerts from the performance pipeline

# How to test?
- Run the fly pipeline command (BUT DON'T APPLY IT!!) to check that no config had inadvertently changed

# Links
- https://trello.com/c/rODkj5dy
